### PR TITLE
Feature/phpstan entities

### PR DESCRIPTION
**Pull request by A-Daneel**
just to check if I can still pass the pipeline.

This should add all the phpstan stuff for my entity test PR (it doesn't as of rn, that's why it's a draft)

### CODE REVIEW

**Comment by A-Daneel** [comment]
I am aware that it does not pass the CI. the coverage should be solved after PR 270 is merged.

Would love some feedback on the parameter types. I've tried to follow the db annotation's, but as you can see from the failing tests that isn't how they are used.

So conformation of what I'm doing (or some feedback on what I've done wrong) would give me some assurances before making this an official PR.

@jasperweyne since you've written most of these entities, I would love to hear your thoughts!

File: src/Entity/Activity/PriceOption.php [hunk 1]
```diff
@@ -48,23 +52,28 @@ class PriceOption
...
     /**
+     * @var int
      * @ORM\Column(type="integer")
      * @GQL\Field(type="Int")
      * @GQL\Description("The price of this option, stored in euro-cents.")
      */
     private $price;
 
     /**
+     * TODO what is it's data type?
```
Comment (jasperweyne) [inline src/Entity/Activity/PriceOption.php:63]: Please formulate todo's in an actionable manner (eg. TODO declare data type)

File: src/Entity/Security/LocalAccount.php [hunk 2]
```diff
@@ -102,10 +113,8 @@ class LocalAccount implements UserInterface, EquatableInterface
 
     /**
      * Get id.
-     *
-     * @return string
      */
-    public function getId(): ?string
+    public function getId(): string
```
Comment (jasperweyne) [inline src/Entity/Security/LocalAccount.php:117]: Id's might be null when they're not stored in the database yet.

File: src/Entity/Security/LocalAccount.php [hunk 3]
```diff
@@ -392,47 +279,19 @@ public function setPasswordRequestedAt(\DateTime $date = null)
...
-     *
-     * @static
-     *
-     * @see
-     * @since
-     */
     public function isPasswordRequestNonExpired($ttl)
     {
         return null === $this->getPasswordRequestedAt() || (
-               $this->getPasswordRequestedAt() instanceof \DateTime &&
```
Comment (jasperweyne) [inline src/Entity/Security/LocalAccount.php:418]: Why is this line removed? The next line assumes the result of `$this->getPasswordRequestedAt()` is not null, and this check is here to make sure of that.

**Review by jasperweyne — CHANGES_REQUESTED** [review]
Couple of comments, also please review [this comment](https://github.com/jasperweyne/helpless-kiwi/pull/270/files#r838978963) in the context of these changes.

File: src/Entity/Security/LocalAccount.php (hunk 3 above)
Comment (A-Daneel) [reply src/Entity/Security/LocalAccount.php:418]: I think this was because phpstan was telling me it would always evaluate to true
`Instanceof between DateTime and DateTime will always evaluate to true.`
Shamefully, I just accepted what the machine told me. Now that I checked it's clear `getPasswordRequestedAt` can also return null.

tldr: line is back
